js-envs/src/main/scala/org/scalajs/jsenv/nodejs/NodeJSEnv.scala

```diff
@@ -40,28 +35,7 @@ class NodeJSEnv private (
   def withSourceMap(sourceMap: Boolean): NodeJSEnv =
     new NodeJSEnv(nodejsPath, addArgs, addEnv, sourceMap)
 
-  /** True, if the installed node executable supports source mapping.
-   *
-   *  Do `npm install source-map-support` if you need source maps.
-   */
-  lazy val hasSourceMapSupport: Boolean = {
-    val code = new MemVirtualJSFile("source-map-support-probe.js")
-      .withContent("""require('source-map-support').install();""")
-
-    try {
-      jsRunner(code).run(NullLogger, NullJSConsole)
-      true
-    } catch {
-      case t: ExternalJSEnv.NonZeroExitException =>
-        false
-    }
-  }
-
   protected def vmName: String = "Node.js"
-  protected def executable: String = nodejsPath
-
-  /** Retry-timeout to wait for the JS VM to connect */
-  protected val acceptTimeout = 5000
```

Comment:
Removing this is not a good idea. It was specifically made protected and overridable at some point. This should be kept here, and that value should used, somehow, from `NodeComJSRunner`